.nowignore

```diff
@@ -0,0 +1,31 @@
+.awcache
+.idea
+.circleci
+.vscode
+coverage
+dll
+docs/dist
+docs/src/componentInfo
+docs/src/componentMenu.json
+docs/src/behaviorMenu.json
+docs/src/currentBundleStats.json
+docs/src/exampleMenus
+docs/src/exampleSources
+node_modules
+packages/**/dist
+packages/**/test
+perf/dist
+
+.editorconfig
+.gitignore
+.prettierignore
+.snyk
+codecov.yml
+dangerfile.ts
+jest.config.js
+lerna-debug.log
+CHANGELOG.md
+LICENSE.md
+MANIFESTO.md
+README.md
+yarn-error.log
```

Comment:
This config allow to ignore files for sync.
General process:
- upload files to Now
- build them with Now
- publish